task3/index.js

```diff
@@ -0,0 +1,81 @@
+// Part A of task 3
+const DOWNLOADS = [
+    {
+      id: 1,
+      title: "Recipe",
+      status: "Done",
+    },
+    {
+      id: 2,
+      title: "Workouts",
+      status: "Pending",
+    },
+    {
+      id: 3,
+      title: "Passwords",
+      status: "Pending",
+    },
+    {
+      id: 4,
+      title: "To Do 2021",
+      status: "Pending",
+    },
+    {
+      id: 5,
+      title: "Books",
+      status: "Failed",
+    },
+  ];
+  
+  document.querySelector(
+    ".content"
+  ).innerHTML = `<table class='downloads__table'></table>`;
+  const renderTable = () => {
+    document.querySelector(".downloads__table").innerHTML = "";
+    for (let i = 0; i < DOWNLOADS.length; i++) {
+      const table = document.createElement("tr");
+      table.innerHTML = `
+      <td> id: ${DOWNLOADS[i].id}</td>
+      <td> title: ${DOWNLOADS[i].title}</td>
+      <td class='td_status'> status: ${DOWNLOADS[i].status}</td>
+      `;
+      document.querySelector(".downloads__table").appendChild(table);
```

Comment:
There is no need to get `.downloads__table` element once again. You can just assign it to the variable and reuse. It will make code much more readable:
```
const table = document.querySelector(".downloads__table");
table..innerHTML = "";
...
table.appendChild(table);
```